packages/integrations/react/server.js

```diff
@@ -23,6 +23,9 @@ async function check(Component, props, children) {
 	}
 	if (typeof Component !== 'function') return false;
 
+	// Preact forwarded-ref components can be functions, which React does not support
+	if (typeof Component === 'function' && Component['$$typeof'].toString() === 'Symbol(react.forward_ref)') return false;
```

Comment:
Two things here:
1. `Component['$$typeof']` might be `undefined` which would cause the `toString()` to throw. That's what's causing the current test failures.
2. Serializing a Symbol to check the stringified value seems like it kinda defeats the purpose of using a `Symbol`? I'm guessing React does not expose the `forward_ref` symbol?

Reply:
Preact exports their symbol here: https://github.com/preactjs/preact/blob/aa53ecb182eb1f5cfff397778150e01b66473f5e/compat/src/forwardRef.js#L13-L17

React, as best I can tell, only exports the symbol from an internal/private shared package here: https://github.com/facebook/react/blob/0cdfef19b96cc6202d48e0812b5069c286d12b04/packages/shared/ReactSymbols.js#L25